src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,21 @@
+$leng-min: 300px;
+$leng-sec: 250px;
+$width-arrows-clock: 10px;
+$leng-circle: $width-arrows-clock * 2;
+
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@mixin circle {
```

Comment:
it would be more flexible to make mixin like this
```
@mixin circle($size) {
  width: $size;
  height: $size;
  border-radius: 50%;
}
```
an use like this:
```
@include circle($leng-circle)
```